x-pack/plugins/security_solution/cypress/objects/rule.ts

```diff
@@ -363,7 +363,11 @@ export const getNewTermsRule = (): NewTermsRule => ({
   mitre: [getMitre1(), getMitre2()],
   note: '# test markdown',
   newTermsFields: ['host.name'],
-  historyWindowSize: getLookBack(),
+  historyWindowSize: {
+    interval: '51000',
```

Comment:
Magic :) I guess you made it > than what `getLookBack()` returns, but it might be not obvious for anyone in the future why we have this specific value here. I think it's worth adding a comment explaining it.